src/main/java/mate/academy/spring/mapper/impl/request/MovieSessionRequestMapper.java

```diff
@@ -7,14 +7,16 @@
 import mate.academy.spring.model.dto.request.MovieSessionRequestDto;
 import mate.academy.spring.service.CinemaHallService;
 import mate.academy.spring.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
The @Autowired annotation is not necessary on the constructor when the class has only one constructor. You can remove it.

Reply:
I used the specific, if you begin use @Autowired you should use it everywhere